arex-instrumentation-api/src/main/java/io/arex/inst/runtime/config/Config.java

```diff
@@ -14,6 +15,7 @@ public class Config {
     private static Config INSTANCE = null;
     private final Map<String, DynamicClassEntity> dynamicEntityMap = new HashMap<>();
 
+    private final Map<String, Set<String>> dynamicClassMethodMap = new HashMap<>();
```

Comment:
Recommended to direct use `getDynamicEntityMap().containskey(buildSignature(method))`